cloudsync_onedrive.py

```diff
@@ -38,6 +39,48 @@
 
 __version__ = "0.1.12"
 
+SOCK_TIMEOUT = 180
+
+session = requests.Session()
```

Comment:
would it make sense to make this an instance variable, and then hang on to the instance of HttpProvider? That would eliminate the need for the global here

Reply:
yeah